.github/workflows/retire.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: erlef/setup-beam@v1
         with:
-          otp-version: '25'
+          otp-version: '26'
           elixir-version: '1.15'
```

Comment:
Don't think this really matters but we use a different version of elixir and otp here than in the rest of the workflows

Reply:
Aligned!